src/components/CrossData.tsx

```diff
@@ -82,14 +103,14 @@ const CrossDataSection = ({
         navigation,
         dataProviderName,
         query,
-        categoryTitle,
+        titleSection,
         categoryId
       )}
       navigation={navigation}
       query={query}
       sectionData={data}
-      sectionTitle={categoryTitle}
-      sectionTitleDetail={categoryTitleDetail}
+      sectionTitle={titleSection}
+      sectionTitleDetail={titleDetail}
```

Comment:
why addings new names instead of reusing the ones existing?

```
sectionTitle={sectionTitle}
sectionTitleDetail={sectionTitleDetail}
```

Reply:
you are right, this is unnecessary. i aligned the naming here